src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/response.tag

```diff
@@ -23,9 +23,9 @@
 </c:if>
 
 <br>
-<div class="form-group margin-0">
-  <div ${isNumResponsesMax ? 'class="col-sm-3 form-inline mobile-align-left"' : 'class="col-sm-5 form-inline mobile-align-left"'}
-      ${isRecipientNameHidden ?  'style="display:none"' : 'style="text-align:right"'}>
+<div class="evalueeForm-${questionWithResponses.question.qnIndx} form-group margin-0" style="padding-left: 75px">
+  <div ${isNumResponsesMax ? 'class="col-sm-12 form-inline mobile-align-left"' : 'class="col-sm-5 form-inline mobile-align-left"'}
```

Comment:
you mean the name evalueeForm ? Well these elements are part of the form for different evaluees that's why ? I think col-sm-12 is useful here since it adds padding to ensure the text box and the recipient name are in line. I tested it. After removing col-sm-12 for second recipient it looks like this:

![image](https://user-images.githubusercontent.com/11634699/40686754-5149470e-6390-11e8-8a6f-d6490514eba1.png)